ecf/include/envir-p1.h

```diff
@@ -15,3 +22,14 @@ fi
 export DBNROOT=$SIPHONROOT
 
 if [[ ! " prod para test " =~ " ${envir} " && " ops.prod ops.para " =~ " $(whoami) " ]]; then err_exit "ENVIR must be prod, para, or test [envir-p1.h]"; fi
+
+export COMROOT=/lfs/h2/emc/ptmp/%EMC_USER%/%PSLOT%/com
+export ROTDIR="$(compath.py -o prod/${NET}/${gfs_ver})"
+if [ -n "%PDY:%" ]; then
+  export PDY=${PDY:-%PDY:%}
+  export CDATE=${PDY}%CYC:%
+fi
```

Comment:
`PDY` is set in the jjobs with `./setPDY.sh`.
Can you explain why this is being set here?

Reply:
This setting sets the PDY on each cycle. That enable the ecflow workflow to run retro parallel.